basic_types.ts

```diff
@@ -657,31 +657,6 @@ export interface Sodium {
     outputFormat: StringOutputFormat,
   ): StringCryptoKX;
 
-  crypto_pwhash(
```

Comment:
So these two method no longer existed on the non sumo sodium for some reason. I only caught this because of the tests. Not sure what else could of been impacted.